README.md

```diff
@@ -265,4 +265,4 @@ several things you should be aware of.
 
 ## 👥 Contribution
 
-See [Code of Conduct](/CODEOFCONDUCT.md).
+See [Contributing](/CONTRIBUTING.md).
```

Comment:
```suggestion
See [Contributing](/CODE_OF_CONDUCT.md).
```

Thanks for pointing out, but it should be like this.

Reply:
Here's an example `CODE_OF_CONDUCT.md` file: https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md

An example `CONTRIBUTING.md` file:
https://github.com/github/docs/blob/main/CONTRIBUTING.md

Based on it's content, it looks like the file should be renamed to `CONTRIBUTING.md`.

Reply:
Oh thanks, I thought they're the same.